runtime/doc/options.txt

```diff
@@ -9638,55 +9638,65 @@ A jump table for the options with a short description can be found at |Q_op|.
+	Completion mode used for the character specified with 'wildchar'.
+	This option is a comma-separated list of up to four parts,
+	corresponding to the first, second, third, and fourth presses of
+	'wildchar'.  Each part is a colon-separated list of completion
+	behaviors, which are applied simultaneously during that phase.
+
+	The possible behavior values are:
+	""		Only complete (insert) the first match.  No further
+			matches are cycled or listed.
+	"full"		Complete the next full match.  Cycles through all
+			matches, returning to the original input after the
+			last match.  If 'wildmenu' is enabled, it will be
+			shown.
+	"longest"	Complete to the longest common substring.  If this
+			doesn't extend the input, the next 'wildmode' part is
+			used.
+	"list"		If multiple matches are found, list all of them.
+	"lastused"	When completing buffer names, sort them by most
+			recently used (excluding the current buffer).  Only
+			applies to buffer name completion.
+	"noselect"	If 'wildmenu' is enabled, show the menu but do not
+			preselect the first item.
+	If only one match exists, it is completed fully—unless "noselect" is
+	specified.
+
+	Some useful combinations of colon-separated values:
+	"longest:full"		Start with the longest common string and show
+				'wildmenu' (if enabled).  Does not cycle
+				through full matches.
+	"list:full"		List all matches and complete first match.
+	"list:longest"		List all matches and complete till the longest
+				common prefix.
+	"list:lastused"		List all matches.  When completing buffers,
+				sort them by most recently used (excluding the
+				current buffer).
+	"noselect:lastused"	Do not preselect the first item in 'wildmenu'
+				if it is active.  When completing buffers,
+				sort them by most recently used (excluding the
+				current buffer).
 
 	Examples: >
 		:set wildmode=full
-<	Complete first full match, next match, etc.  (the default) >
+<	Complete full match on every press (default behavior) >
 		:set wildmode=longest,full
-<	Complete longest common string, then each full match >
+<	First press: longest common substring
+	Second press: cycle through full matches >
 		:set wildmode=list:full
-<	List all matches and complete each full match >
+<	First press: list all matches and complete the first one >
 		:set wildmode=list,full
-<	List all matches without completing, then each full match >
+<	First press: list matches only
+	Second press: complete full matches >
 		:set wildmode=longest,list
-<	Complete longest common string, then list alternatives >
+<	First press: longest common substring
+	Second press: list all matches >
 		:set wildmode=noselect:full
-<	Display 'wildmenu' without completing, then each full match >
+<	Show 'wildmenu' without completing or selecting on first press
+	Cycle full matches on second press >
 		:set wildmode=noselect:lastused,full
-<	Same as above, but sort buffers by time last used.
+<	Same as above, but buffer matches are sorted by last used
```

Comment:
`sorted by last used` doesn't look right. There is no noun following "by".

Reply:
It's understandable, but grammatically iffy. You can change it to "Sorted by time last used", or "Sorted by last use". chrisbra already closed/merged this. Please make this change if you feel strongly about it.

Reply:
@chrisbra

Reply:
I'll update it, thanks both!